yt_dlp/extractor/recurbate.py

```diff
@@ -0,0 +1,44 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from ..utils import (
+    ExtractorError,
+    merge_dicts,
+    update_url_query,
+)
+
+from .common import InfoExtractor
+
+
+class RecurbateIE(InfoExtractor):
+    _VALID_URL = r'https?:\/\/(?:www\.)?recurbate\.com\/play\.php\?video=(?P<id>[0-9]+)'
+    _TEST = {
+        'url': 'https://recurbate.com/play.php?video=39161415',
+        'info_dict': {
+            'id': '39161415',
+            'ext': 'mp4',
+            'title': 'Performer zsnicole33 show on 2022-10-25 20_23, Chaturbate Archive – Recurbate'
+        },
+        'skip': 'Free videos are available for a limited amount of time and for a single session.',
```

Comment:
Since each yt-dlp invocation is a new session, this shouldnt be an issue, correct?

Reply:
I don't know how to explain this but it seems that the free download needs some time to become available again, and by not knowing how and when the tests are executed in this project, I cannot know if the test will be reliable or flaky at best.

Reply:
> needs some time to become available again

Is it IP bound?

Reply:
Multiple runs of the test are passing just fine
```
❯ pytest --tb=short -k Recurb
========================================================================================== test session starts ==========================================================================================
platform win32 -- Python 3.11.0, pytest-7.0.1, pluggy-1.0.0 -- D:\Programs\scoop\apps\python\current\python.exe
cachedir: .pytest_cache
rootdir: D:\Programs\Source\yt-dlp\yt-dlp, configfile: setup.cfg
collected 5249 items / 5247 deselected / 2 selected

test/test_download.py::TestDownload::test_Recurbate PASSED                                                                                                                                         [ 50%]
test/test_download.py::TestDownload::test_Recurbate_all PASSED                                                                                                                                     [100%]

================================================================================== 2 passed, 5247 deselected in 8.96s ===================================================================================

❯ pytest --tb=short -k Recurb
========================================================================================== test session starts ==========================================================================================
platform win32 -- Python 3.11.0, pytest-7.0.1, pluggy-1.0.0 -- D:\Programs\scoop\apps\python\current\python.exe
cachedir: .pytest_cache
rootdir: D:\Programs\Source\yt-dlp\yt-dlp, configfile: setup.cfg
collected 5249 items / 5247 deselected / 2 selected

test/test_download.py::TestDownload::test_Recurbate PASSED                                                                                                                                         [ 50%]
test/test_download.py::TestDownload::test_Recurbate_all PASSED                                                                                                                                     [100%]

================================================================================== 2 passed, 5247 deselected in 4.95s ===================================================================================

```

Reply:
no I believe there's an issue with the test, its downloading and validating a `403 forbidden` page

Reply:
Apologize, I see that this is how these tests are supposed to work, I expected those to download the full videos when running.

Are you sure you can reliably run the test without getting missing cookie exceptions?

Reply:
I am not passing any `cookiefile`. If it breaks when using cookies, it should be considered a bug in the extractor

Reply:
It is not failing when passing cookies, my question was related to how the tests are supposed to work.

My test fails after 3/4 runs